src/components/DiningRoomTable/DiningRoomTable.css

```diff
@@ -0,0 +1,20 @@
+:root {
```

Comment:
You might consider adding the `.module.` part to the names of the css files you created to make the naming consistent with the rest of the code created so far.